app/models/conference_room.rb

```diff
@@ -6,6 +6,7 @@ class ConferenceRoom < ApplicationRecord
   validates :capacity, presence: true
   validates :color, presence: true, uniqueness: { case_sensitive: false }, format: HEX_COLOR_FORMAT
   validates :title, presence: true, uniqueness: true
+  validates :email, presence: true, uniqueness: true
```

Comment:
try to figure out if the uniqueness constraint should be only implemented on the app level or on  the database level to resolve any race condition issues